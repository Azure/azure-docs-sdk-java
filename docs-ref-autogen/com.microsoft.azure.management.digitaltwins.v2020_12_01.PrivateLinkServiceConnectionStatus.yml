### YamlMime:JavaType
uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus"
fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus"
name: "PrivateLinkServiceConnectionStatus"
nameWithType: "PrivateLinkServiceConnectionStatus"
summary: "Defines values for PrivateLinkServiceConnectionStatus."
inheritances:
- "<xref href=\"java.lang.Object\" data-throw-if-not-resolved=\"False\" />"
- "<xref href=\"com.microsoft.rest.ExpandableStringEnum\" data-throw-if-not-resolved=\"False\" />"
inheritedMembers:
- "com.microsoft.rest.ExpandableStringEnum.<T>fromString(java.lang.String,java.lang.Class<T>)"
- "com.microsoft.rest.ExpandableStringEnum.<T>values(java.lang.Class<T>)"
- "com.microsoft.rest.ExpandableStringEnum.equals(java.lang.Object)"
- "com.microsoft.rest.ExpandableStringEnum.hashCode()"
- "com.microsoft.rest.ExpandableStringEnum.toString()"
- "com.microsoft.rest.ExpandableStringEnum.withNameValue(java.lang.String,T,java.lang.Class<T>)"
- "java.lang.Object.clone()"
- "java.lang.Object.finalize()"
- "java.lang.Object.getClass()"
- "java.lang.Object.notify()"
- "java.lang.Object.notifyAll()"
- "java.lang.Object.wait()"
- "java.lang.Object.wait(long)"
- "java.lang.Object.wait(long,int)"
syntax: "public final class PrivateLinkServiceConnectionStatus extends ExpandableStringEnum<PrivateLinkServiceConnectionStatus>"
constructors:
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.PrivateLinkServiceConnectionStatus()"
fields:
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.APPROVED"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.DISCONNECTED"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.PENDING"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.REJECTED"
methods:
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.fromString(java.lang.String)"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.values()"
type: "class"
metadata: {}
package: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
artifact: com.microsoft.azure.digitaltwins.v2020_12_01:azure-mgmt-digitaltwins:1.0.0
