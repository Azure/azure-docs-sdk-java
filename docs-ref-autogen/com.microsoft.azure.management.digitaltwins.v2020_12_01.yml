### YamlMime:JavaPackage
uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
name: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
summary: "This package contains the classes for AzureDigitalTwinsManagementClient.\n Azure Digital Twins Client for managing DigitalTwinsInstance."
classes:
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.AuthenticationType"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.CheckNameRequest"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesPrivateEndpoint"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesPrivateLinkServiceConnectionState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionPropertiesProvisioningState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ConnectionState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResourceProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsIdentity"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsIdentityType"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsPatchDescription"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsPatchProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsResource"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.EndpointProvisioningState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ErrorDefinition"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ErrorResponse"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ErrorResponseException"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.EventGrid"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.EventHub"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ExternalResource"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.GroupIdInformationProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.GroupIdInformationPropertiesModel"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.OperationDisplay"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpoint"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnectionProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ProvisioningState"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PublicNetworkAccess"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.Reason"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.ServiceBus"
interfaces:
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.CheckNameResult"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwins"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.Definition"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.Blank"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithCreate"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithGroup"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithIdentity"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithPrivateEndpointConnections"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.DefinitionStages.WithPublicNetworkAccess"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.Update"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.UpdateStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.UpdateStages.WithIdentity"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsDescription.UpdateStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.Definition"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages.Blank"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages.WithCreate"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages.WithDigitalTwinsInstance"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.DefinitionStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.Update"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.UpdateStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpointResource.UpdateStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.DigitalTwinsEndpoints"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.GroupIdInformation"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.GroupIdInformationResponse"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.Operation"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.Operations"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.Definition"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages.Blank"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages.WithCreate"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages.WithDigitalTwinsInstance"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.DefinitionStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.Update"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.UpdateStages"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnection.UpdateStages.WithProperties"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnections"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateEndpointConnectionsResponse"
- "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkResources"
metadata: {}
package: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
artifact: com.microsoft.azure.digitaltwins.v2020_12_01:azure-mgmt-digitaltwins:1.0.0
