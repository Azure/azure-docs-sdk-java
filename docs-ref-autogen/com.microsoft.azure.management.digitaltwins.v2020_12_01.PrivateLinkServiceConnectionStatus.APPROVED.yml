### YamlMime:JavaMember
uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.APPROVED*"
fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.APPROVED"
name: "APPROVED"
nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
members:
- uid: "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.APPROVED"
  fullName: "com.microsoft.azure.management.digitaltwins.v2020_12_01.PrivateLinkServiceConnectionStatus.APPROVED"
  name: "APPROVED"
  nameWithType: "PrivateLinkServiceConnectionStatus.APPROVED"
  summary: "Static value Approved for PrivateLinkServiceConnectionStatus."
  syntax: "public static final PrivateLinkServiceConnectionStatus APPROVED"
type: "field"
metadata: {}
package: "com.microsoft.azure.management.digitaltwins.v2020_12_01"
artifact: com.microsoft.azure.digitaltwins.v2020_12_01:azure-mgmt-digitaltwins:1.0.0
